packages/react-input/src/components/Input/Input.tsx

```diff
@@ -3,11 +3,12 @@ import { useInput } from './useInput';
 import { renderInput } from './renderInput';
 import { useInputStyles } from './useInputStyles';
 import type { InputProps } from './Input.types';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
 
 /**
  * Input component
  */
-export const Input = React.forwardRef<HTMLSpanElement, InputProps>((props, ref) => {
```

Comment:
Is the HTMLSpanElement no longer needed because InputSlots.root is a span slot?  Or does InputProps need to intersect HTMLSpanElement?